cmd/cmdg/view_messagelist.go

```diff
@@ -961,6 +961,7 @@ func (mv *MessageView) Run(ctx context.Context) error {
 					return nv.Run(ctx)
 				}
 			case "q":
+				screen.Erase()
```

Comment:
I think this clearing is better deferred. That way it can be done in one place even if it's from the openmessage view.

Reply:
Okay sounds good. Just to confirm, do you want to remove this from the `openmessage` or `messagelist` view?

Out of curiosity, are you thinking of eventually incorporating the logic into `MessageViewOp` in some way?

Reply:
I think it's best to have it run on messagelist exit, so defer from there.

I don't think adding it to `MessageViewOp` would be good, since there could be many actions that cause exit (well, at least two, for now), and we want all to clear screen.

Reply:
Ok thanks, I've removed it from the openmessage view now.

Sorry I probably should have just asked directly, could you briefly describe how you plan to go about the multiple clearing? Is it as simple as a defining a function, or more elaborate? This isn't important, but I'm just curious.

Reply:
I mean start the function with something like:

```
defer func(){
   screen.Erase()
   screen.Draw()
}()

Reply:
Thanks, that makes sense. Sorry about the confusion, I am still new to the language and did not realize you were referring to a defer statement. Let me know if it looks okay now.